net-misc/bird/metadata.xml

```diff
@@ -10,6 +10,9 @@
 		<name>Proxy Maintainers</name>
 	</maintainer>
 	<use>
+		<flag name="bmp">
+			Compile with BMP support (alpha support, not prod ready)
```

Comment:
Mask the USE flag then?

Reply:
Sounds good, will do

Reply:
I just did (an reworded the automatic previous commit).
I’m not 100 % sure of me, it’s the first time I’m editing `profiles/base/package.use.mask`